File: GuessMyNumber.java
// package codenomads.examples;

import java.util.Random;
import java.util.Scanner;

public class GuessMyNumber {
    public static void main(String[] args) {
        Random random = new Random();
        int num = random.nextInt(10);

        boolean continuePlaying = true;

        System.out.println("Welcome to GUESS MY NUMBER!");

        while (continuePlaying) {

            System.out.println("Please enter a number between 1 and 10.");
            Scanner scanner = new Scanner(System.in);
            int guess = scanner.nextInt();

            if (guess == num) {
                System.out.println("You win! " + guess + " was the number.");
                continuePlaying = false;
            }

            else if (guess > num) {
                System.out.println("Sorry. Your guess " + guess + " is too high.");
            }
            else if (guess < num) {
                System.out.println("Sorry. Your guess " + guess + " is too low.");
            }
        }
    }
}

File: GuessMyNumber.py
import random

num = random.randint(1, 10)

guess = None

while guess != num:
    guess = input("Guess a number between 1 and 10: ")
    guess = int(guess)

    if guess == num:
        print(f"Congratulations! You won. {num} was the winning number.")
    elif guess > num:
        print("Nope, sorry. Your guess {guess} is too high.")
    elif guess < num:
        print("Nope, sorry. Your guess {guess} is too low.")

